infrastructure/build/fab/lfric_base.py

```
@@ -416,24 +432,12 @@ def get_psyclone_config(self) -> List[str]:
 
     def get_additional_psyclone_options(self) -> List[str]:
         '''
-        :returns: Additional PSyclone command line options. This
-            basic version checks if profiling using Tau or Vernier is enabled,
-            and if so, adds the kernel profiling flags to PSyclone.
-        :rtype: List[str]
+        A placeholder for additional PSyclone comand line options.
```

L425:
•	Another single line method. Is the purpose here to provide points which can be overridden? If so, stating that in the API documentation is important.
•	In this particular case, and if that is the idea, why is there a  {{{get_psyclone_config}}} and a {{{get_additional_psyclone_options}}}?
•	In this particular case, and if that is the idea, why is there a  {{{get_psyclone_config}}} and a {{{get_additional_psyclone_options}}}?
•	Silently failing on file paths which cannot be made relative to the base  path may lead to unexpected behaviour. I would think a warning would be appropriate. Unless there are a lot of non-relative files. What's the reasoning here?
•	Your final return of an empty string does not match your declared return type of {{{Path}}}. Better to return {{{None}}} of type {{{Optional[Path]}}}.